configure.ac

```diff
@@ -4015,12 +4015,6 @@ if test "x$DISABLE_MCS_DOCS" = "xdefault"; then
    DISABLE_MCS_DOCS=$DISABLE_MCS_DOCS_default
 fi
 
-AC_ARG_WITH(lazy_gc_thread_creation, [  --with-lazy-gc-thread-creation=yes|no      Enable lazy runtime thread creation, embedding host must do it explicitly (defaults to no)],[
```

Comment:
It's dead: if you define `LAZY_GC_THREAD_CREATION` then no finalizer thread will ever be created.